### PR TITLE
Init Model

### PR DESCRIPTION
**Pull request by nampt-3121**
## Related Tickets
- [51101](https://edu-redmine.sun-asterisk.vn/issues/51101)

## WHAT (optional)
- Init Model
## HOW
- Add assosication

## WHY (optional)
- Init Model

## Evidence (Screenshot or Video)
Screenshot from 2022-07-19 16-50-56

### CODE REVIEW

**Comment by nampt-3121** [comment]
ready

**Comment by nampt-3121** [comment]
ready

File: app/models/report.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Report < ApplicationRecord
+  belongs_to :from_user, class_name: :User
```
Comment (ngocvt-0484) [inline app/models/report.rb]:
```suggestion
  belongs_to :from_user, class_name: User.name
```

File: db/migrate/20220713085903_create_departments.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateDepartments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :departments do |t|
+      t.string :name
+      t.string :description
```
Comment (ngocvt-0484) [inline db/migrate/20220713085903_create_departments.rb]:
```suggestion
      t.text :description
```

File: db/migrate/20220713090517_create_reports.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class CreateReports < ActiveRecord::Migration[6.0]
+  def change
+    create_table :reports do |t|
+      t.references :from_user, foreign_key: {to_table: :users}
+      t.references :to_user, foreign_key: {to_table: :users}
+      t.references :department, foreign_key: true
+      t.datetime :report_date
+      t.string :today_plan
```
Comment (ngocvt-0484) [inline db/migrate/20220713090517_create_reports.rb]:
```suggestion
      t.text :today_plan
```
các trường today_plan, today_work, today_problem .. nên là trường text nhé

File: db/migrate/20220714034703_create_comments.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, foreign_key: true
+      t.references :report, foreign_key: true
+      t.string     :description
```
Comment (ngocvt-0484) [inline db/migrate/20220714034703_create_comments.rb]:
```suggestion
      t.text     :description
```

**Comment by nampt-3121** [comment]
ready

**Pull request merged**